Cauldron/Assets/Scripts/InputCheatManager.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.InputSystem;
+
+[Serializable]
+public class CheatCodes
+{
+    public string CheatCode;
+    public UnityEvent<string> Event;
+}
+
+public class InputCheatManager : MonoBehaviour
+{
+    [SerializeField]
+    private string compositeString;
+
+    [SerializeField,
+    Range(.1f, 2f)]
+    private float clearAfter = .75f;
+
+    [SerializeField]
+    private List<CheatCodes> cheatCodes;
+
+    private InputAction _anyKeyWait;
```

Comment:
Are you sure you need this action? Seems like you don't do anything with it. Try comment and check.